man/systemd.unit.xml

```diff
@@ -868,18 +868,24 @@
         <term><varname>FailureAction=</varname></term>
         <term><varname>SuccessAction=</varname></term>
 
-        <listitem><para>Configure the action to take when the unit stops and enters a failed state or inactive
-        state. Takes one of <option>none</option>, <option>reboot</option>, <option>reboot-force</option>,
-        <option>reboot-immediate</option>, <option>poweroff</option>, <option>poweroff-force</option> or
-        <option>poweroff-immediate</option>. If <option>none</option> is set, no action will be triggered.
-        <option>reboot</option> causes a reboot following the normal shutdown procedure (i.e. equivalent to
-        <command>systemctl reboot</command>). <option>reboot-force</option> causes a forced reboot which will
-        terminate all processes forcibly but should cause no dirty file systems on reboot (i.e. equivalent to
-        <command>systemctl reboot -f</command>) and <option>reboot-immediate</option> causes immediate execution of the
+        <listitem><para>Configure the action to take when the unit stops and enters a failed state or inactive state.
+        Takes one of <option>none</option>, <option>reboot</option>, <option>reboot-force</option>,
+        <option>reboot-immediate</option>, <option>poweroff</option>, <option>poweroff-force</option>,
+        <option>poweroff-immediate</option>, <option>exit</option>, and <option>exit-force</option>. In system mode,
+        all options except <option>exit</option> and <option>exit-force</option> are allowed. In user mode, only
```

Comment:
this is not quite right, in container/system mode exit is allowed too

Reply:
This is changed in later patch, and then this description is updated too.